### PR TITLE
Improvement_add_Cert_order

### PR DESCRIPTION
**Pull request by ioigoume**
Certificate model add order field

Database change
```sql
alter table cm_certs add column ordr int;
```

### CODE REVIEW

File: app/Model/Cert.php [hunk 1]
```diff
@@ -418,6 +428,65 @@ public function beforeSave($options = array())
...
+      $u = !empty($_SESSION["Auth"]["User"]["username"]) ? $_SESSION["Auth"]["User"]["username"] : null;
+      if(!empty($u)) {
+        $oargs = array();
+        $oargs['joins'][0]['table'] = 'identifiers';
+        $oargs['joins'][0]['alias'] = 'Identifier';
+        $oargs['joins'][0]['type'] = 'INNER';
+        $oargs['joins'][0]['conditions'][0] = 'OrgIdentity.id=Identifier.org_identity_id';
+        $oargs['conditions']['Identifier.identifier'] = $u;
+        $oargs['conditions']['Identifier.login'] = true;
+        // Join on identifiers that aren't deleted (including if they have no status)
```
Comment (lionick) [inline app/Model/Cert.php:446]: Do we need to add a condition for Identifier NOT deleted?

**Review by lionick — CHANGES_REQUESTED** [review]

File: app/Model/Cert.php (hunk 1 above)
Comment (ioigoume) [reply app/Model/Cert.php:446]: This is part of login's query. If it is there i suppose there is a reason. As a result i decided not to experiment.

**Review by lionick — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**